### ISSUE TITLE
Problem safebrowsing and Google

### ISSUE THREAD

**Issue by freeburn12** [mullvad/mullvad-browser#257]
Hi very good job for your fork for free all privacy

Why you haven't deleted all Google and safebrowsing lines: 
```
browser.safebrowsing.provider.google.advisoryURL
browser.safebrowsing.provider.google.lists
browser.safebrowsing.provider.google.reportURL
browser.safebrowsing.provider.google4.advisoryURL
browser.safebrowsing.provider.google4.dataSharingURL
browser.safebrowsing.provider.google4.lists
browser.safebrowsing.provider.google4.reportURL
browser.safebrowsing.provider.google.reportMalwareMistakeURL
browser.safebrowsing.provider.google.reportPhishMistakeURL
```
And you have forgot Mozilla safebrowsing line:

```
browser.safebrowsing.provider.mozilla.lists
browser.safebrowsing.reportPhishURL
```

-And yet you have good erase this line

```
browser.safebrowsing.provider.google.gethashURL
browser.safebrowsing.provider.google4.dataSharingURL
browser.safebrowsing.provider.google4.gethashURL
browser.safebrowsing.provider.mozilla.gethashURL
browser.safebrowsing.provider.mozilla.updateURL
```

And other For Better confidentiality
![gogol mull](https://github.com/mullvad/mullvad-browser/assets/170729697/af0ce4b4-c6e0-4138-bc15-0cf4e2361730)

Please fix for the next update
Thanks

**Comment by Thorin-Oakenpants** [comment]
tor browser and mullvad browser do not utilize google safebrowsing - the prefs are meaningless

**Comment by freeburn12** [comment]
Thanks for your response.

So you could remove these lines to avoid confusion.

Thanks

**Comment by freeburn12** [comment]
So How do you change the internal prefs and settings for an enterprise deployment, without using a .cfg file or policies.json ?

**Comment by Thorin-Oakenpants** [comment]
> So you could remove these lines to avoid confusion

why are you in about:config (rhetorical question)? And no, because it adds complexity and code and rebasing and etc etc where it is not needed

cc: @PieroV who can maybe explain more

**Comment by PieroV** [comment]
> Why you haven't deleted all Google and safebrowsing lines:

Because they aren't active.
They're behind `browser.safebrowsing.*.enabled`, which are set to false.
Also, even if we wanted to use them, we'd have to have an API key which we don't.

I'd be okay with accepting a MR to set them to empty strings, but realistically once the `.enabled` are `false` it's enough for us.

**Comment by freeburn12** [comment]
You could remove these lines to avoid confusion for new users who want to trust you as you say, not like firefox who says that "his priority is our privacy outside GAFAM", while they let the google safebrowsing activate, telemetry ... 

Even if me I know that they are disabled in your fork. A little effort on this would be a must for many people. Please understand it. It's your goal and job.
Thanks

**Comment by PieroV** [comment]
Fair. [I created an upstream issue](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/42603).

**Comment by freeburn12** [comment]
I understand technically, but it is necessary, to be in line with your principles and the trust we have in you. Thank you for your attention

**Comment by ruihildt** [comment]
This has now been fixed with the Mullvad Browser 14.0 release.

See: https://github.com/mullvad/mullvad-browser/releases/tag/14.0
